iothub/device/src/Transport/AmqpIoT/AmqpIoTErrorAdapter.cs

```diff
@@ -34,124 +34,10 @@ internal static class AmqpIoTErrorAdapter
         public static readonly AmqpSymbol ClientVersion = AmqpIoTConstants.Vendor + ":client-version";
         public static readonly AmqpSymbol ApiVersion = AmqpIoTConstants.Vendor + ":api-version";
         public static readonly AmqpSymbol ChannelCorrelationId = AmqpIoTConstants.Vendor + ":channel-correlation-id";
+        public static readonly AmqpSymbol AuthChain = AmqpIoTConstants.Vendor + ":auth-chain";
 
         private const int MaxSizeInInfoMap = 32 * 1024;
 
-        public static AmqpException ToAmqpException(Exception exception, string gatewayId)
-        {
```

Comment:
Why is this removed?

Reply:
These methods were simply lying around in our code, and not being referenced anywhere.

Reply:
They are legacy exception conversion methods that we rewrote when the amqp layer was redesigned, but forgot to remove.